modules/redisearch.ts

```diff
@@ -402,14 +419,16 @@ export class Redisearch extends Module {
      */
     async sugget(key: string, prefix: string, options?: FTSugGetParameters): Promise<string> {
         let args = [key, prefix];
-        if(options !== undefined && options.fuzzy !== undefined)
-            args.push('FUZZY');
-        if(options !== undefined && options.max !== undefined)   
-            args = args.concat(['MAX', options.max.toString()]);
-        if(options !== undefined && options.withScores !== undefined)
-            args.push('WITHSCORES');
-        if(options !== undefined && options.withPayloads !== undefined)
-            args.push('WITHPAYLOADS');
+        if (options !== undefined) {
+            if (options.fuzzy === true)
+                args.push('FUZZY');
+            if (options.max !== undefined)
+                args = args.concat(['MAX', options.max.toString()]);
+            if (options.withScores === true)
+                args.push('WITHSCORES');
+            if (options.withPayloads === true)
+                args.push('WITHPAYLOADS');
+        }
```

Comment:
Why was it bad the way it was before?

Reply:
https://github.com/danitseitlin/redis-modules-sdk-ts/pull/136#discussion_r698296673